crates/paymaster-execution/src/lib.rs

```diff
@@ -78,6 +82,8 @@ pub struct Client {
 
     estimate_account: StarknetAccount,
     relayers: RelayerManager,
+
+    pub diagnostic_service: DiagnosticService,
```

Comment:
I'd rename it as just "diagnostic", less of a mouthful and the "_service" doesn't had more information